src/test/scala/org/broadinstitute/dsde/firecloud/webservice/TrialApiServiceSpec.scala

```diff
@@ -261,6 +261,62 @@ final class TrialApiServiceSpec extends BaseServiceSpec with UserApiService with
     }
   }
 
+  "Campaign manager report endpoints" - {
+
+    "Create Report endpoint" - {
+
+      allHttpMethodsExcept(POST) foreach { method =>
+        s"should reject ${method.toString} method" in {
+          new RequestBuilder(method)("/trial/manager/report") ~> dummyUserIdHeaders("campaignManager") ~> trialApiServiceRoutes ~> check {
```

Comment:
Instead of the `campaignManager` and `nonCampaignManager` `String` literals, consider using `manager` and `unauthorizedUser` defined in `TrialApiServiceSpec` object so we have one place to modify them as necessary.

Reply:
👍  Will do.

Reply:
Ugh, this is tougher. Doing that breaks other tests. Looking more closely into that.

Reply:
Probably not worth it if it ends up taking too much effort.

Reply:
I did get it working ... it was my own stupidity.